examples/differential-expression.md

```diff
@@ -1,23 +1,26 @@
 Find genes differentially expressed by the kidney at significance level 0.05:
+
 ```python
 >>> from os import environ
 >>> from hubmap_api_py_client import Client
 >>> client = Client(environ['API_ENDPOINT'])
 
->>> organ_name = client.select_organs().get_list()[1]['grouping_name']
->>> organ_genes = client.select_genes(where='organ', has=[organ_name], genomic_modality='rna', p_value=0.05)
->>> organ_genes_details = organ_genes.get_list()[0:10]
->>> organ_genes_details[0].keys()
-dict_keys(['gene_symbol', 'go_terms'])
+>>> modality = 'atac'
+>>> gene_symbol = client.select_genes(where='modality', has=[modality]).get_list()[10]['gene_symbol']
+>>> organs_with_gene = client.select_organs(where='gene', has=[gene_symbol], genomic_modality=modality, p_value=1.0)
+>>> organs_with_gene_details = organs_with_gene.get_list()[0:10]
+>>> organs_with_gene_details[0].keys()
+dict_keys(['grouping_name'])
 
 ```
 
-Find organs that differentially express the a gene at the 0.01 significance level
+Find genes differentially expressed by the organ at significance level 1.0:
+
 ```python
->>> gene_symbol = client.select_genes().get_list()[10]['gene_symbol']
->>> organs_with_gene = client.select_organs(where='gene', has=[gene_symbol], genomic_modality='atac', p_value=0.05)
->>> organs_with_gene_details = organs_with_gene.get_list()[0:10]
->>> organs_with_gene_details[0].keys()
-dict_keys(['grouping_name'])
+>>> organ_name = organs_with_gene_details[0]['grouping_name']
```

Comment:
Where are you getting organ names from? We already have a mess of organ terms, and it would be good not to add to the mess. Did you consider using terms from a controlled vocabulary?

Reply:
I've filed
- #73

Reply:
Again, anyone else reading the docs could have the same question.